docs/api/wrappers/supersuit_wrappers.md

```diff
@@ -8,7 +8,7 @@ The [SuperSuit](https://github.com/Farama-Foundation/SuperSuit) companion packag
 
 To convert [space invaders](/environments/atari/space_invaders/) to a greyscale observation space and stack the last 4 frames:
 
-``` python
+``` python notest
 from pettingzoo.atari import space_invaders_v2
```

Comment:
Why’s this one fail?

Reply:
Oh I think it was related to https://github.com/Farama-Foundation/PettingZoo/pull/1075#discussion_r1305759839, I'll take a look at pytest fixtures or another solution

Reply:
Wherever you put this in the CI or locally you should be able to just install the other packages first, better than mock